### PR TITLE
feat: Add PriorityClass in glossary

### PR DESCRIPTION
**Pull request by sergeyshevch**
Hello! This PR address issue #41943. As I see both previous PR is stale and closed. This PR based on #41972 with all comments addressed

### CODE REVIEW

File: content/en/docs/reference/glossary/priority-class.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+aka:
+tags:
+- core-object
+---
+A PriorityClass is a named class for the scheduling priority that should be assigned to a Pod
+in that class.
+
+<!-- more -->
+The PriorityClass is a non-namespaced object mapping a name to an integer priority. The name is 
+specified in the name field, and the priority value in the value field. Priorities range from
```
Comment (sftim) [inline content/en/docs/reference/glossary/priority-class.md:17]:
```suggestion
specified in the `metadata.name` field, and the priority value in the `value` field. Priorities range from
```

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

**Comment by sergeyshevch** [comment]
@sftim Do we need to also add some links to this glossary page somewhere or it will be better to address it in different PR

I didn't found any guideline about glossary shortcode usage

**Comment by natalisucks** [comment]
@sergeyshevch adding links elsewhere to this glossary page can definitely be addressed in a different PR, thanks!

Once the changes Tim has suggested are applied, we can approve this merge

**Comment by sftim** [comment]
(I'm even OK with merging this without the suggested tweak, honestly - it's just good to have the glossary entry)

File: content/en/docs/reference/glossary/priority-class.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+short_description: >
+  A mapping from a class name to the scheduling priority that a Pod should have.
+aka:
+tags:
+- core-object
+---
+A PriorityClass is a named class for the scheduling priority that should be assigned to a Pod
+in that class.
+
+<!-- more -->
```
Comment (dipesh-rawat) [inline content/en/docs/reference/glossary/priority-class.md:15]: The content for this new glossary entry isn't rendering correctly in the [deploy preview (here)](https://deploy-preview-45608--kubernetes-io-main-staging.netlify.app/docs/reference/glossary/?core-object=true#term-priority-class). It seems Hugo is having trouble distinguishing between the short description and the summary content. Consider incorporating the suggested change below to address this, I think it's related to spacing in the comment.

```suggestion
<!--more-->
```

File: content/en/docs/reference/glossary/priority-class.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+aka:
+tags:
+- core-object
+---
+A PriorityClass is a named class for the scheduling priority that should be assigned to a Pod
+in that class.
+
+<!--more-->
+
+[The PriorityClass](/docs/concepts/scheduling-eviction/pod-priority-preemption/#how-to-use-priority-and-preemption) is a non-namespaced object mapping a name to an integer priority. The name is
```
Comment (sftim) [inline content/en/docs/reference/glossary/priority-class.md:17]: (nit)
```suggestion
A [PriorityClass](/docs/concepts/scheduling-eviction/pod-priority-preemption/#how-to-use-priority-and-preemption)
is a non-namespaced object mapping a name to an integer priority, used for a Pod. The name is
```

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by sftim** [comment]
Let's focus on getting this merged.

**Review by dipesh-rawat — COMMENTED** [review]
Glossary entry renders fine in the [preview](https://deploy-preview-45608--kubernetes-io-main-staging.netlify.app/docs/reference/glossary/?core-object=true#term-priority-class) now.

/approve

**Pull request merged**